### DISCUSSION TITLE
我的 2020 🔥软件分享 - Tw93

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#119] [General]
# 我的 2020 🔥软件分享 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share给内部同学分享常用的一些 mac 软件

https://tw93.fun/2020-05-27/good-app.html

**Comment by tw93** [comment]
这篇文章已经时过境迁了，可以看 2022 的这篇 https://tw93.fun/2022-06-25/mac.html
